src/modm/driver/encoder/encoder_slave.hpp

```diff
@@ -0,0 +1,77 @@
+// coding: utf-8
+/*
+ * Copyright (c) 2019, Raphael Lehmann
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+// ----------------------------------------------------------------------------
+
+#ifndef ENCODER_SLAVE_HPP
```

Comment:
```suggestion
#ifndef MODM_ENCODER_SLAVE_HPP
```

avoids name clashing with external code.